src/main/proto/wfa/measurement/internal/kingdom/exchange_step_attempt.proto

```diff
@@ -38,8 +38,30 @@ message ExchangeStepAttempt {
 
   // Which attempt this is. The first attempt should have index 1 and each
   // subsequent attempt should increase the last `attempt_number` by 1.
+  // Output-only.
   int32 attempt_number = 5;
 
   // Additional metadata about the ExchangeStepAttempt.
   ExchangeStepAttemptDetails details = 6;
+
+  enum State {
+    STATE_UNSPECIFIED = 0;
+
```

Comment:
Airflow has the following states, I believe:
```
Dagrun: SUCCESS, RUNNING, FAILED;
Task: SUCCESS, RUNNING, FAILED, UPSTREAM_FAILED, SKIPPED, UP_FOR_RETRY, UP_FOR_RESCHEDULE, QUEUED, NONE, SCHEDULED.
```
I don't want to overthink this but I could see `up_for_retry` and `queued` being helpful status states